tests/logger_logging/test_attributes.py

```diff
@@ -58,7 +58,7 @@ def test_logging_extra_attributes(logger):
     logger.error("extras", extra={"extra_attr": 1})
 
 
-@validate_log_events([{"message": "exc_info"}], required_attrs=["context.exc_info", "context.exc_text"])
+@validate_log_events([{"message": "exc_info"}], required_attrs=["context.exc_info"])
```

Comment:
Leaving these as is because at the time of instrumentation these are None. We determined that these are set in a formatter which is applied after we instrument.